cloudflare/src/endpoints/zone.rs

```diff
@@ -154,7 +157,7 @@ pub struct Zone {
     pub status: Status,
     /// An array of domains used for custom name servers. This is only available for Business and
     /// Enterprise plans.
-    pub vanity_name_servers: Vec<String>,
+    pub vanity_name_servers: Option<Vec<String>>,
```

Comment:
Why make this an Option? Why not just pass an empty vector?

Reply:
These fields that I changed to `Option` weren't returned at all in the response body, which was causing Serde to fail to deserialize it. Making them `Option`s instead of providing defaults keeps the deserialized structure true to the actual contents of the response.